sway/input/seatop_move_tiling.c

```diff
@@ -127,27 +201,53 @@ static void handle_motion_postthreshold(struct sway_seat *seat) {
 		return;
 	}
 
+	// Check if the cursor is over a tilebar only if the destination
+	// container is not a descendant of the source container.
+	if (!surface && !container_has_ancestor(con, e->con) &&
+			split_titlebar(node, e->con, cursor->cursor,
+			&e->drop_box, &e->after_target)) {
```

Comment:
Nit: I think we want an extra tab here for the hanging params of `split_titlebar`.